### PR TITLE
gdk-pixbuf -- adding recipe

### PR DESCRIPTION
**Pull request by notestaff**
Adding recipe for gdk-pixbuf, a library for image loading and manipulation from GNOME.

### CODE REVIEW

File: recipes/gdk-pixbuf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    - toolchain
+    - gcc  # [linux]
+    - llvm  # [osx]
+    - python <3
+    - m4
+    - autoconf
+    - automake
+    - libtool
+    - pkg-config
+    - glib >=2.48.0
```
Comment (scopatz) [inline recipes/gdk-pixbuf/meta.yaml]: This should probably be pinned to the standard value: `glib 2.51.*`

https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L49

File: recipes/gdk-pixbuf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - gobject-introspection
+    - libffi
+    - libtiff
+    - libpng
+    - perl
+    - gettext ==0.19.7  # [osx]
```
Comment (scopatz) [inline recipes/gdk-pixbuf/meta.yaml]: Is this pin really needed?

**Comment by scopatz** [comment]
Thanks for the PR @notestaff!

**Comment by notestaff** [comment]
@scopatz pins you mentioned are now fixed

**Comment by scopatz** [comment]
thanks!

**Pull request merged**